content/r-package-dev/Version_Control.md

```diff
@@ -3,7 +3,7 @@ author: Lindsay R. Carr
 date: 9999-10-31
 slug: git
 title: Version Control
-draft: True
+draft: FALSE 
```

Comment:
Did you check locally if this work? by running Hugo without the build draft option?